src/app.js

```diff
@@ -19,6 +52,18 @@ db.sequelize.sync()
     console.log('Error syncing with database', err);
   });
 
+/**
+ * @swagger
+ * /:
+ *   get:
+ *     summary: gets hello world
+ *     tags: [Hello]
+ *     responses:
+ *       200:
+ *         description: Success
+ *       500:
+ *         description: Server error
+ */
```

Comment:
I don't like you to use this format of documentation, because they make codes look like supergetti; please use documentation in the format of .js; here is the reference you can look into. [Link](https://github.com/atlp-rwanda/Dev-Pulse-BN/blob/develop/src/swagger/index.js)

Reply:
Fixed!